apps/podium-service/chalicelib/repository/repository.py

```diff
@@ -19,6 +19,28 @@
 #     return True
 
 
+def add_projects(projects: list[data_types.Projects]) -> Optional[str]:
+    def add(session: Session) -> str:
+        to_insert = []
+        verticals = get_verticals()
+        for project in projects:
+            to_insert.append(
+                models.Project(
+                    vertical_id=verticals[verticals.index(project.vertical)],
```

Comment:
This isn't quite correct: project.vertical returns the name of the vertical, which doesn't work with verticals.index() since verticals is a list of vertical objects. verticals.index() also returns the index of the vertical in the list (which can be arbitrarily-ordered), whereas we want the vertical ID which is UUIDv4.